skbeam/core/roi.py

```diff
@@ -398,50 +398,55 @@ def mean_intensity(images, labeled_array, index=None):
     return mean_intensity, index
 
 
-def circular_average(image, calibrated_center, threshold=0, nx=100,
-                     pixel_size=(1, 1),  min_x=None, max_x=None):
+def circular_average(image, r_array, pixel_size=(1, 1), statistic='mean',
```

Comment:
this will be a new function @CJ-Wright maybe change the name now to avoid confusion?